sycl/doc/extensions/USM/USM.adoc

```diff
@@ -558,69 +680,56 @@ Return value:: none
 
 '''
 === Unified Shared Memory Information and Descriptors
-
-
-.Unified Shared Memory Pointer Query
+==== Pointer Queries
+===== get_pointer_type
 [source,cpp]
 ----
-std::tuple<usm::alloc, sycl::device*> get_pointer_info(const void* ptr);
+usm::alloc get_pointer_type(const void *ptr, const context &ctxt);
 ----
 Parameters::
   * `const void* ptr` - the pointer to query
-Return value:: Returns a `std::tuple` containing the type of the allocation and a pointer to the `device` against which it was allocated.  If this is a host allocation, the `device` pointer will be `nullptr`.
+  * `const sycl::context& ctxt` - the SYCL `context` to which the USM allocation belongs
+Return value:: Returns the USM allocation type for the allocation pointed to by `ptr`.  If `ctxt` is a host `context`, returns `usm::alloc::host`.
+
+===== get_pointer_device
+[source,cpp]
+----
+sycl::device get_pointer_device(const void *ptr, const context &ctxt);
+----
+Parameters::
+ * `const void* ptr` - the pointer to query
+ * `const sycl::context& ctxt` - the SYCL `context` to which the USM allocation belongs
+ Return value:: Returns the `device` associated with the USM allocation.  If `ctxt` is a host `context`, returns the host `device` in `ctxt`.  If `ptr` is an allocation of type `usm::alloc::host`, returns the first device in `ctxt`.
```

Comment:
Same questions about `ptr` restrictions as `get_pointer_type`.

Reply:
Will likely throw an error if the ptr was not a USM allocation.  Need to double check.